sources/packages/api/src/esdc-integration/e-cert-integration/e-cert-full-time-integration/e-cert-full-time-integration.service.ts

```diff
@@ -65,8 +64,20 @@ export class ECertFullTimeIntegrationService extends ECertIntegrationService {
     // Calculated values.
 
     const fileRecords = ecertRecords.map((ecertRecord) => {
+      let filterAwards = ecertRecord.awards;
+      if (ecertRecord.stopFullTimeBCFunding) {
+        const bcAwards = [
+          DisbursementValueType.BCTotalGrant,
+          DisbursementValueType.BCLoan,
+        ];
+        filterAwards = ecertRecord.awards.filter(
+          (disbursementValue) =>
+            !bcAwards.includes(disbursementValue.valueType),
```

Comment:
a simple `not equal check` should be fine here, IMO.are we expecting `bcAwards` list to grow?

Reply:
No, there are only loans and grants. As we are having 2 values, this way it will be cleaner.